[FEATURE] Add extra info for the image component (#2383)
magento/pwa-studio: pwa-devdocs/src/frequently-asked-questions/index.md

    -   Add `headers:{ Store: YOUR_STORE_CODE }`

> ## Image component loads wrongly sized images from the srcSet

Reviewer: ```suggestion
## Image component loads images with the wrong size from the srcSet
```

Author: This adjustment wont pass the 60 char limit for headers I believe

Reviewer: ah, you're right. we can leave as is then

> When you are using the `<Image/>` component from `venia-ui` you need to make sure to have changed the following values:

Reviewer: ```suggestion
When you use the `<Image/>` component from `venia-ui`, change the following values:
```

-   [packages/venia-ui/lib/util/images.js][]

    -   Change `DEFAULT_WIDTH_TO_HEIGHT_RATIO` to match your image ratio.
    -   Change the values in the `imageWidths` mapping to better fit your dimensions.

-   [packages/venia-ui/lib/components/gallery/item.js][]

>     -   Make sure to supply the `<Image />` component with a `widths` prop as shown in this example.

Reviewer: ```suggestion
    -   Supply the `<Image />` component with a `widths` prop as shown in the GalleryItem component defined in the linked file.
```

{: .bs-callout .bs-callout-info}

> _**Note:** The native device emulator in Chrome will give you incorrect values, it's best to resize the viewport._

Reviewer: ```suggestion
_**Note:** For testing, resize the viewport manually instead of using the native device emulator in Chrome, which gives incorrect values._
```
